Formula/minipro.rb

```diff
@@ -27,7 +28,11 @@ def install
 
   test do
     output_minipro = shell_output("#{bin}/minipro 2>&1", 1)
-    assert_match "minipro version #{version}", output_minipro
+    if build.head?
+      assert_match "minipro version", output_minipro
+    else
+      assert_match "minipro version #{version}", output_minipro
+    end
```

Comment:
Since we don't run CI on head builds we can keep this as
```suggestion
    assert_match "minipro version #{version}", output_minipro
```